test/functional/test-resources.js

```diff
@@ -1824,6 +1833,67 @@ describe('Resources mutateElement and collapse', () => {
     });
   });
 
+  it('attemptCollapse should not call attemptChangeSize', () => {
```

Comment:
We're targeting the [`scrollAdjSet`](https://github.com/ampproject/amphtml/blob/12c9c61/src/service/resources-impl.js#L1015-L1042) path, right?

If so, we should be able to avoid the `getScrollHeight` stub, and just assert that `resource1#completeCollapse` has ben called before `resource1#getLayoutBox`.